raccoglitesi.go

```diff
@@ -150,7 +151,10 @@ func getDocenti(codiceDipartimento string) []Docente {
 			nome := strings.TrimSpace(link.Text())
 			ruolo := infoBlock.Find("p").First().Text()
 			if !exists {
-				log.Fatal("La pagina dei docenti è probabilmente cambiata, non posso prendere trovare il link alla sua pagina")
+				fmt.Fprintln(os.Stderr,
+					"La pagina dei docenti è probabilmente cambiata, "+
+						"non posso prendere trovare il link alla sua pagina")
```

Comment:
Correggere il doppio verbo "prendere trovare".
La congiunzione con virgola (_sentence-splice_) è un errore
in italiano. Magari usare i due punti anziché la virgola
(sì, so che era già così).